routes/index.js

```diff
@@ -1,20 +1,34 @@
 import express from 'express'
 import passport from 'passport'
 const router = express.Router()
+import { getRandomlyPlaylist } from '../controllers/playlistController'
+import { getSongById, updateSongPlays } from '../controllers/songController'
+import { getAlbumById, getAlbums } from '../controllers/albumController'
 import {
   validateSignup,
   signUp,
   signIn,
   getCurrentUser,
   requestReset,
   resetPassword,
-  fbSignIn
+  fbSignIn,
```

Comment:
This comma `,` at the end seems unnecessary. Use Linter for JS to catch this kind of things